pkg/gather/clusterconfig/clusterconfig.go

```diff
@@ -63,8 +63,24 @@ func (i *Gatherer) Gather(ctx context.Context, recorder record.Interface) error
 			}
 			data, err := i.metricsClient.Get().AbsPath("federate").
 				Param("match[]", "ALERTS").
+				// aggregator metrics indicate how apiservers are performing as viewed from the kube-apiserver
+				Param("match[]", "aggregator_unavailable_apiservice_count").
+				Param("match[]", "aggregator_unavailable_apiserver_gauge").
```

Comment:
do these have alerts?

Reply:
> do these have alerts?

Not sure actually.  Not in the kas-o where I would expect them to be defined.